apps/ewallet_db/lib/ewallet_db/transaction_consumption.ex

```diff
@@ -189,6 +236,15 @@ defmodule EWalletDB.TransactionConsumption do
     end
   end
 
+  def get_final_amount(consumption) do
```

Comment:
I think I need a `@spec` for this one. I'll probably forget why we need it a few weeks from now.